package.json

```diff
@@ -36,11 +36,11 @@
   },
   "devDependencies": {
     "assemble-contrib-i18n": "0.1.*",
+	"autoprefixer": "^8.2.0",
```

Comment:
Indent

Reply:
Ugh, sorry. I'll fix it in a sec.

Recently got a new workstation and completely forgot to reinstall my editor's EditorConfig plugin.